src/bin/systemd-crontab-generator.cpp

```diff
@@ -643,22 +624,6 @@ struct Job {
 		this->schedule += ":00"sv;
 	}
 
-	auto generate_scriptlet() -> std::optional<std::string> {
```

Comment:
please keep this one part.

the part that expand the pgm name with which() will go.